ts/change-notetype/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "include": ["*"],
     "references": [
         { "path": "../lib" },
+        { "path": "../lib/keys" },
```

Comment:
This section lists typescript projects, not individual files, so this line is not required.